modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -4,6 +4,24 @@ definition {
 		Click(locator1 = "AppBuilder#BUTTON_ADD");
 	}
 
+	macro addApp {
```

Comment:
Hey Brooke, I was planning to create a macro called Deploy.macro that we will add everything related to the Apps tab, so the additional macros selectForm, selectTableView will be also inside of Deploy.macro. 

So we will keep the AppBuilderAdmin macro as a General/Common library containing actions that are available to others parts of the App Builder, like the actions of deleteEntry that you created that is applicable to FormViews, TableViews and Apps tabs.